packages/polaris-viz/src/utilities/getFontSize.ts

```diff
@@ -0,0 +1,9 @@
+import {FONT_SIZE, MOBILE_FONT_SIZE} from '@shopify/polaris-viz-core';
+
+export function getFontSize() {
+  const isMobile = 'ontouchstart' in window || navigator.maxTouchPoints > 0;
```

Comment:
For some of the other mobile changes, we might watch to separate the mobile check into its own util/hook, separate from the font size but it doesn't need to be on this PR